homeassistant/components/moehlenhoff_alpha2/sensor.py

```diff
@@ -0,0 +1,95 @@
+"""Support for Alpha2 base and IO device sensors."""
+import logging
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import Alpha2BaseCoordinator
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Add Alpha2 sensor entities from a config_entry."""
+
+    coordinator: Alpha2BaseCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    async_add_entities(
+        Alpha2IODeviceBatterySensor(coordinator, io_device_id)
+        for io_device_id, io_device in coordinator.data["io_devices"].items()
+        if io_device["_HEATAREA_ID"]
+    )
+    # HEATCTRL attribute ACTOR_PERCENT is not available in older firmware versions
+    async_add_entities(
```

Comment:
Instead of calling `async_add_entities` twice, create a list and pass that into a single `async_add_entities` call

Reply:
Okay.